src/model/ModelBase.js

```diff
@@ -413,6 +413,22 @@ export default class ModelBase {
   }
 
   /**
+   * @returns {Object}
+   */
+  @memoize
```

Comment:
memoize is not needed here since we simply return a value. Memoizing this will only make it slower